Servers/driver/autoDriver.driver.ts

```
@@ -439,15 +439,15 @@ export async function insertMockData() {
-      projects![0].id,
-      projects![1].id
+      vendors![0].id!,
+      vendors![1].id!,
+      vendors![2].id!,
+      vendors![3].id!,
+      projects![0].id!,
+      projects![1].id!
     ).map((d) => vendorsProjectsGenerateValuesString(d as any));
     insertString += values.join(",") + ";";
-    vendorsProjects = await insertData(insertString as string);
+    vendorsProjects = (await insertData(insertString as string) as [{}[], number])[0];
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)